### PR TITLE
Fix #9155: updated the error messages for duplicate entity failure for better context

### LINKED ISSUE #9155: entity already exists
**Affected module**
Register a new account screen

**Describe the bug**
During a first step I created a first account with the same  first- and lastname together with a gmail address. Upon click "Create account" I got a message saying that the account has been created and the account needs to be activated via the activation mail.

During a second step I created a second account with the same first- and lastname together with a hotmail address. When clicking on "Create account" I get the error message `entity already exists`

**To Reproduce**

1. Navigate to http://localhost:8585
2. Click on Create Account
3. Fill in the form (for the first time)

- First Name: Example
- Last Name: Demo
- Email: demo@gmail.com

4. Click on Create Account
5. (Normally you should receive a message that the account has been created)
6. Create another (second) account

- First Name: Example
- Last Name: Demo
- Email: demo@hotmail.com

7. Click on Create Account
8. Now you should receive the error `entity already exists`

**Expected behavior**

**Version:**
 - OS: [e.g. iOS] Windows 10 machine
 - Python version: 3.10.8
 - OpenMetadata version: [e.g. 0.8] latest version running in a docker container
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes:

Fixes #9155 

I worked on updating the error messages for duplicate entity failures for following entities:
 

- [x] User
- [x]  Team
- [x]  Query
- [x]  Bot
- [x]  Service
- [x]  Glossary
- [x]  Tag
- [x]  Roles
- [x] Policy
- [x]  Rules
- [x]  Alert
- [x]  Test case
- [x]  Test suite
- [x]  Classification

Screenshot 2023-06-02 at 4 34 00 PM
Screenshot 2023-06-02 at 4 34 29 PM
Screenshot 2023-06-02 at 4 34 54 PM
Screenshot 2023-06-02 at 4 35 30 PM
Screenshot 2023-06-02 at 4 38 26 PM
Screenshot 2023-06-02 at 4 39 04 PM

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by ShaileshParmar11** [comment]
image
this is not looking right can you check?

**Comment by aniketkatkar97** [comment]
@ShaileshParmar11 Actually, the thing is when a user is created, `name` is extracted from the email itself. Example: for mail id 'aniket@gmail.com' the name will be 'aniket'.

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx [hunk 1]
```diff
@@ -163,7 +164,24 @@ const AddDataQualityTestV1: React.FC<AddDataQualityTestProps> = ({
       setActiveServiceStep(3);
       setTestCaseRes(testCaseResponse);
     } catch (error) {
-      showErrorToast(error as AxiosError);
+      if (
+        (error as AxiosError).response?.status === HTTP_STATUS_CODE.CONFLICT
+      ) {
+        showErrorToast(
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx:170]: @aniketkatkar97 , Can we directly handle the conflict case in the `showErrorToast` method? like we are handling for status codes 401 and 403.

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx:170]: No, Don't want to make `showErrorToast` lengthy

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**